pkg/volume/gce_pd/gce_util.go

```diff
@@ -146,7 +146,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (volum
 		return "", 0, err
 	}
 
-	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB))
+	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB), *c.options.CloudTags)
```

Comment:
I need to add a new item into `VolumeOptions`, which would be ignored by the other clouds. It looks better to me if I implement its handling for all clouds in one PR.